tests/array-copy/Makefile.in

```diff
@@ -0,0 +1,2 @@
+TOP_FILE := $(TEST_DIR)/top.sv
```

Comment:
This test will now run in Verilator as well, so it needs a *.cpp file (or at least an issue filed to add it later).

Reply:
This test is skipped for Verilator in CI for now: https://github.com/antmicro/verilator/blob/add-ci/.github/workflows/main.yml#L65

Reply:
OK, I see they're skipped in vcddiff tests in Yosys as well.